src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -554,11 +554,16 @@ CreateDistributedTable(Oid relationId, Var *distributionColumn, char distributio
 	{
 		List *partitionList = PartitionList(relationId);
 		Oid partitionRelationId = InvalidOid;
+		Oid namespaceId = get_rel_namespace(relationId);
```

Comment:
Sorry that we don't have a documentation describing how we typically do the backport.  We try to avoid merge commits on the backport, so avoid pull requests to release branches.

Instead, what we roughly follow is:

- checkout to release branch on your local machine && fetch the latest.
     -  I typically do `git fetch -a && git checkout release-XXX && git reset --hard origin/release-XXX` to make sure my local is no different than the origin for the `release-XXX`
- cherry pick the relevant commits to this release branch locally
- checkout to another branch, say `release-XXX-teja`
- push the new branch to origin, and make sure all the tests pass 
- Once all the tests pass, checkout back to the `release-XXX` and push it to the origin


We also repeat this on the citus-enterprise as well. But, before cherry-picking the changes to the enterprise repo, first you need to get the citus-community changes to citus-enterprise. You can see the instructions for that here: https://github.com/citusdata/citus-enterprise/blob/enterprise-master/CONTRIBUTING.md#merging-community-changes-onto-enterprise

Once the citus community changes are merged to citus-enteprise, follow the above steps to update the release branch of the enterprise repository as well.

@tejeswarm hope this is clear, let us know if anything is unclear, feel free to raise any questions on the engine-chat to get faster response.

Reply:
Done. Thanks for the detailed notes.